src/Data/Formatter/DateTime.purs

```diff
@@ -212,16 +227,27 @@ padQuadrupleDigit i
   | i < 1000 = "0" <> (show i)
   | otherwise = show i
 
+-- | Format a DateTime according to the format defined in the given `Formatter`
 format :: Formatter -> DT.DateTime -> String
 format f d = foldMap (formatCommand d) f
 
+-- | Format a DateTime according to the format defined in the given format string.
+-- | If the format string is empty, or contains astral plane characters (i.e., unicode 
+-- | code points that aren't representable in a single code unit), will return a `Left` value. 
```

Comment:
I could use a spot check on this statement.  I originally thought that only empty strings could produce a `Left` value, but checked that with QuickCheck, and got "ᆌH졶簮" as another value of an input that produces `Left`.  I'm pretty sure this is because those are astral plane characters that aren't representable via `Char`, which is used in `placeholderContent` up on 132.  But honestly, I'm not the most familiar with unicode and code units vs. code points.

Reply:
I feel like I should fix the parser to support a better unicode range rather than having to do documentation gymnastics to excuse it :smile:. Let's remove the statement, and I'll take a look at fixing it.

Reply:
Hah! It took a minute to figure it out, as I was going down the unicode track, but the reason that input fails is because of the `H` in there - it's a partial formatter command. 

The parser it's expecting `HH`, so throwing a failure when it fails to see another `H` after the first one.

Reply:
🤦‍♂️ 
ugh, that's embarrassing.  Okay I'll update this.